api/metrics-api-jaxrs-1.1/src/main/java/org/hawkular/metrics/api/jaxrs/handler/MetricHandler.java

```diff
@@ -82,6 +88,41 @@
     @HeaderParam(TENANT_HEADER_NAME)
     private String tenantId;
 
+    @POST
+    @Path("/")
+    @ApiOperation(value = "Create metric definition.", notes = "Clients are not required to explicitly create "
+            + "a metric before storing data. Doing so however allows clients to prevent naming collisions and to "
```

Comment:
+1, very nice wording in the notes. That said, I think we create a metric, not a metric definition. "Creating a metric definition" sounds a bit confusing to me.